samples/wifi/sta/README.rst

```diff
@@ -179,21 +179,40 @@ To measure the current consumption of the nRF5340 SoC, complete the following st
  * Remove jumper on **P22** (VDD jumper).
  * Cut solder bridge **SB16**.
  * Make a short on solder bridge **SB17**.
- * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.
-    You can use **P21** pin **1** mentioned as **GND** (-).
- * Connect the **Vout** on PPK2 to **P22** pin **1** on the nRF7002 DK.
+ * Connect **GND** on PPK2 kit to **GND** on the nRF5340 SoC.
+
+     You can use **P4** pin **7** mentioned as **GND** for ground.
+
+     Note that this connection requires a berg pin connector.
+
+ * Connect the **Vout** on PPK2 to **P22** pin **1** on the nRF5340 SoC.
```

Comment:
Replace nRF5340 with nRF7002 DK

Reply:
No, this was intentional, DK covers both 5340 SoC + 7002 device, so, to clarify we are using individual component names.

Reply:
Sorry, after re-reading, I understand your intention, will fix.